### PR TITLE
fix(product-template): adjust spacing and styles

### PR DESCRIPTION
**Pull request by raff-r**

### CODE REVIEW

**Review by endymion1818 — APPROVED** [review]

File: src/components/templates/ProductTemplate/ProductTemplateProgress/ProductTemplateProgress.tsx [hunk 1]
```diff
@@ -3,12 +3,18 @@ import styled from 'styled-components';
 
 import Progress, { ProgressProps } from '../../../molecules/Progress/Progress';
```
Comment (admmasters) [inline src/components/templates/ProductTemplate/ProductTemplateProgress/ProductTemplateProgress.tsx:5]: Nit: is the extra spacing needed?

File: src/components/templates/ProductTemplate/ProductTemplateCard/ProductTemplateCard.tsx [hunk 2]
```diff
@@ -12,13 +11,11 @@ interface ProductTemplateCardProps {
...
-  ${maxMedia.phone`
-    ${css`
-      border-radius: 0;
-      box-shadow: 0 0 1px 0 ${colors.greyLight}, 0 0 -1px 0 ${colors.greyLighter};
-      border: none;
-    `}
-  `}
+  @media (max-width: ${grid.breakpoints.m}px) {
+    border-radius: 0;
+    box-shadow: 0 0 1px 0 ${colors.greyLight}, 0 0 -1px 0 ${colors.greyLighter};
```
Comment (rominamoya) [inline src/components/templates/ProductTemplate/ProductTemplateCard/ProductTemplateCard.tsx:16]: hi @raff-r  could you check  this value for box-shadow as it shows as invalid

**Review by rominamoya — APPROVED** [review]
:100:

**Review by admmasters — APPROVED** [review]

**Pull request merged**